frontend/packages/policy-editor/README.md

```diff
@@ -0,0 +1,3 @@
+# policy-editor
+
+More information will be added while working on the package
```

Comment:
Since this package has features implemented already, should we add a little more information here? :)